chart/hyrax/templates/_helpers.tpl

```diff
@@ -86,6 +86,23 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- end }}
 {{- end -}}
 
+
+{{/*
+Create default fully qualified service names.
```

Comment:
should these be used below in the templates?

Reply:
hyrax.fits.fullname is uses in fits.host. fits.host is used in configmap-env.yaml line 38

Reply:
ohh, i see.